### ISSUE TITLE
Running in dev mode breaks proxy-portlet and embedding API

### ISSUE THREAD

**Issue by ajw625** [orbeon/orbeon-forms#2119]
When I switched the `oxf.run-mode` from `prod` to `dev` it caused some display issues. These occur only when using the proxy-portlet or embedding API. See attached images for embedding. The same issues occur with the proxy-portlet.

![embedding_display_issues_01](https://cloud.githubusercontent.com/assets/9143532/6470632/d946c56a-c199-11e4-92e6-26dbe2a3c929.png)
![embedding_display_issues_02](https://cloud.githubusercontent.com/assets/9143532/6470631/d9454d66-c199-11e4-97a7-d67aa752edea.png)

**Comment by ebruchez** [comment]
This is because in dev mode, we set `oxf.xforms.combine-resources` to `false`. But the proxy portlet and embedding currently require `oxf.xforms.combine-resources` to `true` so that they can perform URL rewriting.

As a workaround, you can set in your properties:

```xml
 <property
    as="xs:boolean"
    name="oxf.xforms.combine-resources"
    value="false"/>
```

**Comment by ajw625** [comment]
Thanks. I guess you meant set it to false. I added this to my properties and it fixed the issue:

```xml
    <property
        as="xs:boolean"
        name="oxf.xforms.combine-resources"
        value="true"/>
```

**Comment by ebruchez** [comment]
Yes I meant `true`, I fixed it soon after posting the comment ;)
